main.js

```diff
@@ -235,24 +235,20 @@ function addRemoveActiveNav(nav1, nav2) {
 }
 
 function filterByMerchant(merchantId) {
-  const specificMerchantItems = []
+  return items.filter((item) => 
+    item.attributes.merchant_id === parseInt(merchantId))
+}
 
```

Comment:
I am looking forward to seeing this functionality implemented on the page and the +/- differences, if any.